src/components/TitleCard/index.js

```diff
@@ -2,16 +2,18 @@ import React from 'react'
 import {Heading, Paragraph, Card} from 'egghead-ui'
 
 export default ({children, title, description}) => (
-  <section className='mb4'>
-    <Heading level='4'>
-      {title}
-    </Heading>
-    {description
-      ? <Paragraph type='small'>
-          {description}
-        </Paragraph>
-      : null
-    }
+  <section className='mb4 br2 bg-white-secondary shadow-1'>
+    <div className='pa4'>
+      <Heading level='5'>
+        {title}
+      </Heading>
+      {description
+        ? <Paragraph type='small'>
+            {description}
+          </Paragraph>
+        : null
+      }
+    </div>
```

Comment:
Updates the title card to put the title/description inside the card with latest mock styles:
![image](https://cloud.githubusercontent.com/assets/5497885/24430136/6aa76b86-13d2-11e7-8f68-3f376c1c1b38.png)